engine/lib/views.php

```diff
@@ -1269,6 +1269,57 @@ function elgg_view_form($action, $form_vars = array(), $body_vars = array()) {
 }
 
 /**
+ * Renders a form field
+ *
+ * @tip Field vars contain both field and input params. 'label', 'help', and 'field_class' params will
```

Comment:
I think this tip should be the inline doc for the $vars param. We should also indicate "label" and "help" accept HTML (they will not be escaped on output).